### PR TITLE
Separate index file for assets

### PR DESCRIPTION
**Pull request by RoiEXLab**
## Overview
This PR moves all of the asset location references to a dedicated file.
@DanVanAtta Is it intended that all the resources land in the game-headed/assets directory?
I doubt that you forgot to add a `.gitignore` file.

## Functional Changes

[ ] New map or map update
[ ] New Feature
[ ] Feature update or enhancement
[ ] Feature Removal
[x] Code Cleanup or refactor
[ ] Configuration Change
[ ] Bug fix

## Testing

[ ] Covered by existing automated tests
[ ] Covered by newly added automated tests
[x] Manually tested
[ ] No testing done

### CODE REVIEW

File: game-headed/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+assets
```
[the remark this answers is not in the document]
Comment (RoiEXLab) [reply game-headed/.gitignore]: I had about 170 files that weren't ignored after running the Headed game config, triggering gradle.
Just checked the file extensions: gif, mp3, txt

File: game-headed/assets.txt [hunk 2]
```diff
@@ -0,0 +1,950 @@
+# To generate the below list, clone the assets repo, then run:
```
[the remark this answers is not in the document]
Comment (RoiEXLab) [reply game-headed/assets.txt]: Good point

[the remark this answers is not in the document]
Comment (RoiEXLab) [reply game-headed/assets.txt]: Having `.assets.txt` is good enough IMO

**Comment by DanVanAtta** [comment]
@RoiEXLab I'll go ahead and take care of the .gitignore updates and submit those for a PR

**Pull request merged**